cmd/antrea-agent/config.go

```diff
@@ -111,15 +111,21 @@ type AgentConfig struct {
 	// "udp" L4 transport protocols.
 	// Defaults to "flow-aggregator.flow-aggregator.svc:4739:tcp".
 	FlowCollectorAddr string `yaml:"flowCollectorAddr,omitempty"`
-	// Provide flow poll interval in format "0s". This determines how often flow exporter dumps connections in conntrack module.
-	// Flow poll interval should be greater than or equal to 1s(one second).
-	// Defaults to "5s". Follow the time units of duration.
+	// Provide flow poll interval in format "0s". This determines how often flow
+	// exporter dumps connections in conntrack module. Flow poll interval should
+	// be greater than or equal to 1s(one second).
+	// Defaults to "5s". Follow the time units of duration type.
 	FlowPollInterval string `yaml:"flowPollInterval,omitempty"`
-	// Provide flow export frequency, which is the number of poll cycles elapsed before flow exporter exports flow records to
-	// the flow collector.
-	// Flow export frequency should be greater than or equal to 1.
-	// Defaults to "12".
-	FlowExportFrequency uint `yaml:"flowExportFrequency,omitempty"`
+	// Provide active flow export timeout, which is a timeout for each active flow
```

Comment:
could you update the comments in this file to match the yaml ones?

Reply:
Done.